source/configure/site-configuration-settings.rst

```diff
@@ -252,12 +252,22 @@ Forgot Password custom link
   This configuration setting applies to all Mattermost clients, including web, desktop app, and mobile app. You can control whether the **Forgot Password** link is visible or hidden in clients by going to **Authentication > Password > Enable Forgot Password Link**. See the :ref:`configuration <configure/authentication-configuration-settings:enable forgot password link>` documentation for details.
 
 
+.. config:setting:: report-a-problem-type
+  :displayname: Report a Problem type (Customization)
+  :systemconsole: Site Configuration > Customization
+  :configjson: .SupportSettings.ReportAProblemType
+  :environment: MM_SUPPORTSETTINGS_REPORTAPROBLEMTYPE
+  :description: Select how the ‘Report a Problem’ option behaves. Choosing ‘Custom link’ or ‘Email address’ allows you to provide a URL or address in the next field. ‘Hide link’ removes the ‘Report a Problem’ option from the app.
```

Comment:
There is another option (Default) that will use the permalinks that we have discussed before
```
https://mattermost.com/pl/report_a_problem_licensed -> https://support.mattermost.com/hc/en-us/requests/new
https://mattermost.com/pl/report_a_problem_unlicensed -> https://handbook.mattermost.com/contributors/contributors/ways-to-contribute#report-a-bug
```